tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=86
+deps =
+    coverage
+
+[testenv:flake8]
+basepython = python3.6
+skip_install = true
+deps = flake8
+commands = flake8 --doctests src setup.py
```

Comment:
Personally I prefer `[flake8] doctests = 1` in a setup.cfg (or your favourite config file from the multitude that flake8 supports), so that people who have editor/IDE integration with flake8 don't need to tweak it not to get false positive warnings from imports used only in doctests and such.